glyphs-reader/src/font.rs

```diff
@@ -566,8 +519,17 @@ impl Font {
 }
 
 impl Font {
-    pub fn read_glyphs_file(glyphs_file: &path::Path) -> Result<Font, Error> {
+    pub fn load(glyphs_file: &path::Path) -> Result<Font, Error> {
         let raw_content = fs::read_to_string(glyphs_file).map_err(Error::IoError)?;
+
+        // Glyphs has a wide variety of unicode definitions, not all of them parser friendly
+        // Make unicode always a string, without any wrapping () so we can parse as csv, radix based on format version
+        let re = Regex::new(
+            r#"(?m)^(?P<prefix>\s*unicode\s*=\s*)[(]?(?P<value>[0-9a-zA-Z,]+)[)]?;\s*$"#,
+        )
+        .unwrap();
+        let raw_content = re.replace_all(&raw_content, r#"$prefix"$value";"#);
```

Comment:
clever hack.. but it's a pity we have to resort to this. The v3 formatting of unicodes was meant to overcome this ambiguity, so treating all as csv strings feels to me like a push back. But if it's the price to pay to keep supporting v2, I can live with that.

Reply:
> The v3 formatting of unicodes was meant to overcome this ambiguity

To be fair if we only cared about v3 it would, v3 the only issue is that sometimes it's a scalar and sometimes a sequence. This is the simplest way I could think of to deal with it and I think the simplicity has some value.

Reply:
well, that's the way it is, in v3 the type is either a scalar or array of scalar, which one can deal with, and a generic parser can work with without extra hacks like this regex find/replace.
Also, now that newlines are no longer escaped with octal escape sequences but added verbatim, I believe (but not 100% certain, need to test) that it's not unthinkable to have multiline strings (e.g. note or FEA features) containing that `unicode = {digits};` pattern may occur and you'd be incorrectly replacing that..

Reply:
Theoretically true I think. We could reduce the odds by including the exact amount of preceding whitespace (Glyphs app seems to be consistent) or make it fully sound by altering the approach.

Filed https://github.com/googlefonts/fontmake-rs/issues/40 as I think this is sufficient to bootstrap us.